### ISSUE TITLE
Change to billing/node(s) resets node pool limit count

### ISSUE THREAD

**Issue by vidiecan** [Shippable/support#4501]
behaviour:
- we have one default node pool with 4 nodes, node limit set to 4
- updated billing information (new card added)
- found that node limit was reset to 1

the same behaviour happened when we did the following (we found out only after the last step but the culprit might be in any step)
- created new node pool with 1 node
- deleted new node pool
- added node pool to the default pool

**Comment by vidiecan** [comment]
(we are not going to reproduce so feel free to close the issue)
